### DISCUSSION TITLE
Lokinet & Session don't connect on Android

### DISCUSSION THREAD

**Discussion by mahdi-ln** [oxen-io/lokinet#2007] [General]
Lokinet & Session don't connect on Android phone. Session can get connected by 3rd-party VPN.
Desktop versions works without need of 3rd-party VPN.

Country: Iran

**Comment by majestrate** [comment]
i am guessing that they block `seed.lokinet.org` which the super old lokinet apk build uses to bootstrap from.

**Reply by mahdi-ln** [reply]
Session: 1.15.4
Lokinet: Lokinet-android-0.9.9.apk is used (It shows 0.9.8 as version)

**Reply by majestrate** [reply]
that apk is known to not work at all. use the lokinet-android-0.9.8.apk

**Comment by mahdi-ln** [comment]
Someone said ISPs block connections based on the device OS. But I don't recommend mimicking desktop connection as they will block it altogether

**Reply by majestrate** [reply]
if the isp knows the device os, it is possible they have access to the device to get this info.
i do not think that an os block is happening as lokinet wire protocol looks the same (uniform random data in udp packets) regardless of the os.
